build/x86_64_centos6/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM centos:6
+
+RUN yum update -y && yum install -y \
+	automake \
+	autconf \
+	bison \
+	epel-release \
+	expect \
+	flex \
+	gcc \
+	git \
+	java-devel \
+	java-1.6.0-openjdk-devel \
+	libcurl-devel \
+	libtool \
+	make \
+	mysql-devel \
+	openssl-devel \
+	perl-devel \
+	postgresql-devel \
+	python-devel \
+	rpm-build \
+	which \
+	yajl-devel
+
+RUN yum install -y \
```

Comment:
Any reason why this isn't either a) included with the previous dependencies, or b) chained with the previous run command to prevent another layer?

Reply:
Done and done.